pkg/scheduler/apis/default_plugins.go

```diff
@@ -0,0 +1,136 @@
+/*
+Copyright 2022 The Clusternet Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package apis
+
+import (
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/klog/v2"
+
+	"github.com/clusternet/clusternet/pkg/scheduler/framework/plugins/names"
+)
+
+// getDefaultPlugins returns the default set of plugins.
+func getDefaultPlugins() *Plugins {
+	return &Plugins{
+		PreFilter: PluginSet{},
+		Filter: PluginSet{
+			Enabled: []Plugin{
+				{Name: names.TaintToleration},
+			},
+		},
+		PostFilter: PluginSet{},
+		PrePredict: PluginSet{},
+		Predict: PluginSet{
+			Enabled: []Plugin{
+				{Name: names.Predictor},
+			},
+		},
+		PreScore: PluginSet{},
+		Score: PluginSet{
+			Enabled: []Plugin{
+				{Name: names.TaintToleration, Weight: 3},
+			},
+		},
+		PreAssign: PluginSet{
+			Enabled: []Plugin{
+				{Name: names.DynamicAssigner},
+			},
+		},
+		Assign: PluginSet{
+			Enabled: []Plugin{
+				{Name: names.StaticAssigner},
+				{Name: names.DynamicAssigner},
+			},
+		},
+		Reserve: PluginSet{},
+		Permit:  PluginSet{},
+		PreBind: PluginSet{},
+		Bind: PluginSet{
+			Enabled: []Plugin{
+				{Name: names.DefaultBinder},
+			},
+		},
+		PostBind: PluginSet{},
+	}
+}
+
+// mergePlugins merges the custom set into the given default one, handling disabled sets.
```

Comment:
Same for this.
We should add a reference link and say this piece of codes are copied from somewhere.